stock_product_pack/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module allows you to get the right available *On Hand* and *Forecasted*
+quantities of the packs.
```

Comment:
Bad english 
_This module allows you to get On Hand and Forecasted values calculated from components of the pack._